packages/modal-ui/src/lib/components/styles.css

```diff
@@ -902,21 +929,16 @@
 }
 
 .nws-modal-wrapper .nws-modal .nws-modal-body .alert-message .connection button {
-    margin: 24px auto;
+    padding: 10px 24px;
+    gap: 8px;
+    background: rgba(79, 124, 209, 0.16);
```

Comment:
Please try to set color/background-color through css variables in the future, it will be easier to switch them and manage the styles.